packages/inputs-minimal/src/password_field/index.jsx

```diff
@@ -1,9 +1,9 @@
 import { useState } from 'react';
-import PropTypes from 'prop-types';
-import TextField from '../textfield';
-import omit from 'lodash/omit';
 import Eye1Icon from '@goodhood/icons/lib/medium/eye_1';
 import Eye2Icon from '@goodhood/icons/lib/medium/eye_2';
+import omit from 'lodash/omit';
+import PropTypes from 'prop-types';
```

Comment:
by purpose also re-sort lodash and prop-types (from npm)?

Reply:
I was mainly reordering `TextField` to be further down, will push these 2 to the top, looks kind of weird below Eye1Icon and Eye2Icon